packages/presentation/lib/src/pages/home_page/home.dart

```diff
@@ -39,56 +44,52 @@ class _HomeState extends BlocScreenState<Home, HomeBloc> {
           )
         ],
       ),
-      bottomNavigationBar: BottomNavigationBar(
```

Comment:
I recommend to implement clicking on the bottomnavigationbar, I recommend using indexedstask for this